### ISSUE TITLE
php artisan cashier:install now working anymore, and no config/cashier_coupons.php file

### ISSUE THREAD

**Issue by basiltap** [laravel/cashier-mollie#304]
I'm trying to install molly in laravel 8 but no way to use "php artisan cashier:install". I receive an error:  There are no commands defined in the "cashier" namespace.

Also, looking for config/cashier_coupons.php file, there is no any cashier config file.

**Comment by sandervanhooft** [comment]
What have you tried so far?

**Comment by basiltap** [comment]
Today I installed it in a fresh laravel app, and it worked. But I found the error when I was trying to install it in an app with a previous cashier intallation for Stripe.

I followed the documentation:
- MOLLIE_KEY="test_xxxxxxxxxxx"   (and cleared the cache)
- composer require laravel/cashier-mollie "^1.0"
- php artisan cashier:install   (and here I found the error from Artisan: There are no commands defined in the "cashier" namespace. ) and, of course, any config file was created, ...
